src/python/pants/backend/python/rules/run_setup_py.py

```diff
@@ -491,7 +491,7 @@ def _is_exported(target: HydratedTarget) -> bool:
   """
   hydrated_target = owned_dependency.hydrated_target
   ancestor_addrs = AscendantAddresses(hydrated_target.address.spec_path)
-  ancestor_tgts = await Get[HydratedTargets](Specs, Specs((ancestor_addrs,)))
+  ancestor_tgts = await Get[HydratedTargets](AddressSpecs((ancestor_addrs,)))
```

Comment:
You lost one argument in that function call. Just making sure it was intentional?

Reply:
Yes, I'm using the shorter syntax for `Get`. You can either say:

Reply:
Pleasure. For next time, maybe better to do 2 commits; one with plain rename and one with small refactor to explain intent to reviewer (I will know for this, but maybe another noob reviews or a similar situation occurs with a different case :) )